README.md

```diff
@@ -102,7 +102,13 @@ you run the playbook:
 $ ansible-playbook -i inventory/vms.local.generated kube-install.yml -e 'container_runtime=crio'
 ```
 
-Additionally, the compilation of CRI-O requires a beefier machine, memory-wise. It's recommended you spin up the machines with 4 gigs of ram or greater. During the VM creation phase, should you use it. One may wish to add the parameters `-e "ram_mb=4096" ` to your playbook run of `virt-host-setup.yml`.
+Additionally, the compilation of CRI-O requires a beefier machine, memory-wise. It's recommended you spin up the machines with 4 gigs of ram or greater. During the VM creation phase, should you use it. One may wish to add the parameters `-e "vm_parameters_ram_mb=4096"` to your playbook run of `virt-host-setup.yml`.
+
+## Using Fedora
+
+You might consider using Fedora for this setup, especially with CRI-O at this time. Especially as Buildah requires functionality in later Kernels that, at the time of writing this section of the readme, is only available in Fedora.
```

Comment:
I'm not sure I'm in love with the "You might consider using..." type phrasing in the documentation. Can we try something more assertive?

Use of Fedora is currently suggested when using CRI-O. Buildah requires functionality in later Linux kernels that are unavailable in recent versions of CentOS.